highfidelity/hifi
# Fixed: Avatar hovers off ground while running

This PR addresses the hovering of avatars while running.  Case https://highfidelity.manuscript.com/f/cases/21176

A T-Pose to the beginning of the  default running animation.

### Test Plan
1) open hifi in desktop mode. go into 3rd person and put the camera so that you are at floor level with a view of the avatar feet.  
2) press shift forward arrow to run.
  
Expected Result:  The avatar feet should not raise up off the ground during the run
Repeat this with http://hifi-content.s3.amazonaws.com/angus/rootHipsBug/robot_arm_coincident_spineChest/robotarm/avatar.fst
